course_content/cartopy_course/solutions/cartopy_exercise_3

```diff
@@ -0,0 +1,41 @@
+
+# Cartopy course, exercise 3
+
+# Use set_extent to control plot : what coordinate system does this use ?
+
+# Repeat the 'correct' plot with a latlon-straight line, but control the map region.
+proj = ccrs.EquidistantConic()
+x0, y0 = -50, -30
+x1, y1 = 10, 55
+
+# Extend corners slightly
+ax = plt.axes(projection=proj)
+ax.coastlines()
+
+plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())
+
+# Use "set_extent", with slight margins added to the region corners.
+x0a = x0 - 0.1 * (x1 - x0)
+x1a = x1 + 0.1 * (x1 - x0)
+y0a = y0 - 0.1 * (y1 - y0)
+y1a = y1 + 0.1 * (y1 - y0)
```

Comment:
The margins, while nice, seem a bit more complication than is needed.

Reply:
Ok, let's use fixed numbers instead.  I guess that will be clear enough.